src/apps/activities/api/activities.py

```diff
@@ -358,3 +361,109 @@ async def __filter_activities(
                     activities.remove(activity)
 
     return activities
+
+
+async def applet_activities_counters_for_subject(
+    applet_id: uuid.UUID,
+    subject_id: uuid.UUID,
+    user: User = Depends(get_current_user),
+    language: str = Depends(get_language),
+    session=Depends(get_session),
+    answer_session=Depends(get_answer_session),
+) -> Response[ActivitiesCounters]:
+    applet_service = AppletService(session, user.id)
+    await applet_service.exist_by_id(applet_id)
+    await CheckAccessService(session, user.id).check_applet_detail_access(applet_id)
+
+    await SubjectsService(session, user.id).exist_by_id(subject_id)
```

Comment:
We need to exclude limited accounts from auto-assigned activities. It's a straightforward fix. Part 1:

```suggestion
    subject = await SubjectsService(session, user.id).exist_by_id(subject_id)
    is_limited_respondent = subject.user_id is None
```